### PR TITLE
Package template merge

### PR DESCRIPTION
**Pull request by keflavich**
I squashed all of package-template into one commit and merged it.  There's one open question, though.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -78,6 +78,14 @@
 package_info['package_data'].setdefault(PACKAGENAME, [])
 package_info['package_data'][PACKAGENAME].append('data/*')
 
+# Adam's note: I don't know what to do with this.
+# # Define entry points for command-line scripts
+# entry_points = {}
+# entry_points['console_scripts'] = [
+#     'astropy-package-template-example = packagename.example_mod:main',
+# ]
```
Comment (keflavich) [inline setup.py]: @astrofrog: can you tell me what ought to be done with this?  Should `packagename` be changed to `astroquery` here?  Should `example_mod` also be astroquery?  I'm unfamiliar with this.  Maybe this is only for command line scripts, which astroquery currently does not include?

Comment (keflavich) [inline setup.py]: @cdeil since you were helping with package-template-merging stuff, do you have any idea what I should be doing with this segment?

**Comment by keflavich** [comment]
This probably replaces #470

File: setup.py (hunk 1 above)
Comment (astrofrog) [inline setup.py]: You can just remove all that because there are no scripts in astroquery.

Comment (keflavich) [inline setup.py]: OK, thanks - I just wanted to confirm that this was indeed for scripts.  I'll change the comment.

**Pull request merged**